### DISCUSSION TITLE
.md file img src URLs should all be relative and not absolute

### DISCUSSION THREAD

**Discussion by rgsiiiya** [willwulfken/MidJourney-Styles-and-Keywords-Reference#27] [Ideas and Style Submissions]
Being able to clone the repo locally for super fast browsing of the docs would be incredibely beneficial.  Cloning is trivial, and it's a reasonable size for all the images.

But, looking at the .md files, they all use absolute img src URLs.  So even when you clone the repo, and use a viewer, it still pulls all the images down from github.

Would be awesome if all the img src URLs were relative so that they would work locally.

Does github support relative URLs in md files?

**Comment by willwulfken** [comment]
GitHub does support this, but I have had complications with it for this particular project in the past

The docs would have to be gone through, regexed, and reviewed. This is something I thought about doing before, but I never did since I started looking for a better way

**Comment by willwulfken** [comment]
Here you go

https://github.com/willwulfken/MidJourney-Styles-and-Keywords-Reference/tree/relative-links

**Comment by hot2hot** [comment]
Hello. I have generated image https://i.postimg.cc/j5BWrkYc/photo-2023-03-01-15-40-10.jpg
I need to make exactly the same, so that only the lion is red. How can I do that?

And then the picture is constantly changing completely. Help me please. I won't be in debt)

**Reply by rgsiiiya** [reply]
This is probably not the correct place for your question, but I'll try to answer.

--seed is essential for repeatability in midjourney.  If you did not specify your own --seed, MJ generates one for you.  You can get that --seed by sharing your image to yourself via the DM option.

Search YouTube for variations of "consistent characters" and you will find lots of helpful vids for how do this.

**Reply by willwulfken** [reply]
You can also try giving MidJourney one of those images and typing your original prompt with "red" at the beginning
